### PR TITLE
Update/csi

### PR DESCRIPTION
**Pull request by hlwanghl**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Additional documentation e.g., usage docs, etc.**:

```docs

```

### CODE REVIEW

File: ansible/roles/app-role-k8s/files/opt/app/current/bin/node/k8s.sh [hunk 1]
```diff
@@ -440,16 +442,21 @@ setUpNodeLocalDns() {
 }
 
 setUpStorage() {
+  # Remove previous versions
   local csiNode; for csiNode in $(getColumns $INDEX_NODE_INSTANCE_ID $STABLE_MASTER_NODES $STABLE_WORKER_NODES); do
     runKubectlDelete -n kube-system csinode $csiNode
```
Comment (MrYourM) [inline ansible/roles/app-role-k8s/files/opt/app/current/bin/node/k8s.sh:447]: 该报错导致创建资源失败
```
kubectl --kubeconfig /data/kubernetes/admin.conf get -n kube-system csinode i-b8vyycnj --no-headers -oname
Error from server (NotFound): csinodes.storage.k8s.io "i-b8vyycnj" not found
```
实际 csinode
```
$ kubectl --kubeconfig /data/kubernetes/admin.conf get -n kube-system csinode --no-headers -oname
csinode.storage.k8s.io/master1
csinode.storage.k8s.io/worker-p001
```

Comment (hlwanghl) [reply ansible/roles/app-role-k8s/files/opt/app/current/bin/node/k8s.sh:447]: ok，改成 getNodeNames 了

File: ansible/roles/app-role-k8s/files/opt/app/current/bin/node/k8s.sh [hunk 2]
```diff
@@ -440,16 +446,22 @@ setUpNodeLocalDns() {
...
   runKubectlDelete -n kube-system ds csi-qingcloud-node
-  local -r csiDefaultFile=/opt/app/current/conf/k8s/csi-qingcloud-$QINGCLOUD_CSI_VERSION.yml
-  local -r qingcloudCfgDefaultFile=/opt/app/current/conf/qingcloud/config.default.yaml
-  yq r -d18 $csiDefaultFile data[config.yaml] > $qingcloudCfgDefaultFile
-  yq w -d18 $csiDefaultFile data[config.yaml] -- "$(yq m -a $QINGCLOUD_CONFIG $qingcloudCfgDefaultFile)" | runKubectl apply -f -
+
+  # CSI plugin
+  local -r csiChartFile=/opt/app/current/conf/k8s/csi-qingcloud-$QINGCLOUD_CSI_VERSION.tgz
+  local -r csiValuesFile=/opt/app/current/conf/k8s/csi-qingcloud-values.yml
+  yq p $QINGCLOUD_CONFIG config | cat - $csiValuesFile | \
```
Comment (MrYourM) [inline ansible/roles/app-role-k8s/files/opt/app/current/bin/node/k8s.sh:461]: Error: Kubernetes cluster unreachable

**Comment by MrYourM** [comment]
> /assign @MrYourM

LGTM

**Review by MrYourM — APPROVED** [review]
/lgtm

**Pull request merged**